src/main/data/99_version.sql

```diff
@@ -95,3 +95,4 @@ INSERT INTO version (version) VALUES ('2.21.0:20180419.01');
 INSERT INTO version (version) VALUES ('2.21.0:20180426.01');
 INSERT INTO version (version) VALUES ('2.21.0:20180515.01');
 INSERT INTO version (version) VALUES ('2.22.0:20180523.01');
+INSERT INTO version (version) VALUES ('2.23.0:20180601.01');
```

Comment:
`v2.21.0` conversions were just released in prod.
Shouldn't this conversion still be `v2.22.0`?